teps/0079-tekton-catalog-support-tiers.md

```diff
@@ -100,14 +128,467 @@ requirements, and provenance attestations can be made to meet software supply ch
+Tekton Catalog Maintainers to migrate resources from the Community Catalog
+to the Official Catalog.
+
+#### Community and Official Catalogs in the Tekton Hub
+
+Users rely on the [Tekton Hub][hub] to discover shared resources. 
+The Tekton Hub supports publishing resources from multiple Catalogs.
+As described in [TEP-0003][tep-0003-hub], the goal was to set it up to
+support providing support tiers for shared resources. It also allows
+users and organizations to create their own Catalogs and share them in
+the Tekton Hub, as long as they comply with the Catalog contract. The
+Tekton Hub can indicate the source Catalog of each resource, such as
+through a badge or tag. 
+
+To add the Official Catalog, we only have to modify the Catalogs 
+[configuration][hub-config] in the Hub, as such:
+
+```yaml
+catalogs:
+  
+  # community catalog
+  - name: tekton-community
+    org: tektoncd
+    type: community
+    provider: github
+    url: https://github.com/tektoncd/catalog
+    revision: main
+    
+  # official catalog  
+  - name: tekton-official
+    org: tektoncd
+    type: official
+    provider: github
+    url: https://github.com/tektoncd/catalog-official
+    revision: main
+```
+
+This provides extensibility to the support tiers: we can add additional
+Catalogs for other support tiers as needed in the future. 
+
+#### Community and Official Catalogs in the Tekton CLI
+
+Today, users can use the CLI to install resources from Catalogs by
+passing in the Catalog name to the `--from` argument as shown in the
+examples below. This will add a label to the resource indicating the
+source Catalog: `hub.tekton.dev/catalog=<catalog-name>`.
+
+```shell
+# Community Catalog
+
+$ tkn hub install task golang-lint --version 0.3 --from tekton-community
+
+Task golang-lint(0.3) installed in default namespace
+
+$ kubectl describe task.tekton.dev/golang-lint
+
+Name:         golang-lint
+Namespace:    default
+Labels:       app.kubernetes.io/version=0.3
+              hub.tekton.dev/catalog=tekton-community
+...
+ 
+# Official Catalog
+
+$ tkn hub install task golang-fuzz --version 0.1 --from tekton-official
+
+Task golang-fuzz(0.1) installed in default namespace
+
+$ kubectl describe task.tekton.dev/golang-fuzz
+
+Name:         golang-fuzz
+Namespace:    default
+Labels:       app.kubernetes.io/version=0.1
+              hub.tekton.dev/catalog=tekton-official
+...
+```
+
+#### Community and Official Catalogs in the Cluster
+
+When resources are installed in a cluster, without using the CLI, 
+it may be difficult to identify which Tekton Catalog it came from
+because they won't have the labels added by the CLI.
+
+To make it easy for users to identify the source Catalog, we propose 
+adding a `tekton.dev/catalog` annotation to the resource yaml file, 
+as such:
+
+```yaml
+# resource from the community catalog
+annotations:
+  tekton.dev/pipelines.minVersion: "0.17.0"
+  tekton.dev/tags: golang-lint
+  tekton.dev/catalog: community
```

Comment:
NIT: perhaps instead of a name we could use the URL of the catalog? That way it would be easier to know where a task comes from, as organisations may be using more catalogs than official and community.

Reply:
makes sense, updated this in https://github.com/tektoncd/community/pull/613